packages/react/src/components/CTA/CTA.js

```diff
@@ -136,6 +347,7 @@ const _renderButtons = ({ buttons }) =>
  * @returns {*} object
  */
 const _renderFeatureCard = featureCard => {
+  if (featureCard.type === 'video') featureCard.cta.href = '#';
```

Comment:
Would this run into issues if there were actual anchor links on the page? Like if the link was inside of the Table of Contents?

Reply:
Resolving, looks like `preventDefault` will take care of this.